source/NanostackRfPhyAtmel.cpp

```diff
@@ -1709,6 +1708,9 @@ static void rf_start_tx(void)
     rf_flags_set(RFF_TX);
     /*RF state change: SLP_TR pulse triggers PLL_ON->BUSY_TX*/
     rf_if_enable_slptr();
+    /*Chip permits us to write frame buffer while it is transmitting*/
+    /*As long as first byte of data is in within 176us of TX start, we're good */
```

Comment:
Did you measure this 176us or is it from datasheet? Does it matter which datarate is used?

Reply:
It's from the RF233 datasheet, It's the preamble time at 250kbits.

I'm assuming it's okay, because it's comparable to the ack turnaround time. We manage to switch to RX mode within the needed 192us (@250kbit) of getting the TX_END interrupt, and that includes interrupt response time, thread switching, and more SPI commands.

So this 176us when we're already running for 1 buffer write should be no issue at all.

You have reminded me about the RF212 though - this has had basically no testing on it.